app/streaming/video/ffmpeg-renderers/d3d11va.cpp

```diff
@@ -455,6 +855,47 @@ bool D3D11VARenderer::initialize(PDECODER_PARAMETERS params)
         return false;
     }
 
+    // Check if the GPU is capable of AI-Enhancement
+    // This capability setup is place in this method because it is only available on FFmpeg with DirectX for hardware acceleration
+    m_VideoEnhancement->enableVideoEnhancement(false);
+    if(m_VideoEnhancement->isEnhancementCapable()){
+
+        // Enable the visibility of Video enhancement feature
+        m_VideoEnhancement->enableUIvisible();
+
+        StreamingPreferences streamingPreferences;
+        if(streamingPreferences.videoEnhancement){
```

Comment:
You shouldn't construct a `StreamingPreferences` because it will not contain variables set via the command-line. To pass this into the renderers, you should add it to the `DECODER_PARAMETERS` struct.

Reply:
Done